### DISCUSSION TITLE
Printing whitespace characters other than space/tab

### DISCUSSION THREAD

**Discussion by mgagic** [olikraus/u8g2#2249] [Q&A] [answered]
I am using u8g2_font_profont11_tf and I would like to use thin space or one of the other narrower white spaces. Is there a way to do it with this font?

**Comment by olikraus** [comment, marked as answer]
Maybe I missed this question...  Usually there is only space (unicode 32) in the fonts. Any other whitespace could be implemented by creating a custom font.

**Comment by mgagic** [comment]
No worries, thanks anyway :)
